Misc/Test-Suite/StefanStyle/runTests.pl

```diff
@@ -32,12 +32,12 @@ sub getPath {
 if ((not defined $subTask) or ($subTask eq "default")) {
 	# runs for approx. 9min ...
 	checkPseudoknotMFEPP("pseudoknots.fasta", "pseudoknots mfe*pp pknotsRG",   "-s P -P $Settings::RNAPARAM1999", "pseudoknots.fasta.mfepp.pknotsRG.out") ;
-	checkPseudoknotMFEPP("pseudoknots.fasta", "pseudoknots mfe*pp strategy A", "-s A -P $Settings::RNAPARAM1999", "pseudoknots.fasta.mfepp.pKissA.out") if ((not defined $subTask) or ($subTask eq "knots"));
-	checkPseudoknotMFEPP("pseudoknots.fasta", "pseudoknots mfe*pp strategy B", "-s B -P $Settings::RNAPARAM1999", "pseudoknots.fasta.mfepp.pKissB.out") if ((not defined $subTask) or ($subTask eq "knots"));
-	checkPseudoknotMFEPP("pseudoknots.fasta", "pseudoknots mfe*pp strategy C", "-s C -P $Settings::RNAPARAM1999", "pseudoknots.fasta.mfepp.pKissC.out") if ((not defined $subTask) or ($subTask eq "knots"));
-	checkPseudoknotMFEPP("pseudoknots.fasta", "pseudoknots mfe*pp strategy D", "-s D -P $Settings::RNAPARAM1999", "pseudoknots.fasta.mfepp.pKissD.out") if ((not defined $subTask) or ($subTask eq "knots"));
-	checkParameters("pseudoknots parameter check", $TMPDIR."/".$Settings::ARCHTRIPLE.'/'.$Settings::PROGINFOS{'pkiss'}->{name}."_mfe", "pseudoknots.parametercheck.out") if ((not defined $subTask) or ($subTask eq "knots"));
-	checkBasicFunctions("basic pseudoknot functions", "pseudoknots.basic.out") if ((not defined $subTask) or ($subTask eq "knots"));
```

Comment:
Is the if-condition really no longer required? Or is there any case in which it could be relevant and will lead to complications now that it's missing?

Reply:
I think I messed this condition up while learning how to split tests for Travis. I wanted to make sure that all tests are executed without a parameter and if this parameter is given and thus defined only specific tests should be run. I must have planned with "default", "shapes", "pkiss" AND "knots" but than saw by looking at execution times that the overhead for another "knots" test was larger than the extra time spent by these tests.
In master, these lines can be reached if variable `$subTask` is not defined, but never if set to `knots`.